scripts/build.py

```diff
@@ -287,6 +294,30 @@ def _compare_file_count(first_dir_path, second_dir_path):
                 first_dir_file_count, second_dir_file_count))
 
 
+def _compare_file_count_for_multiple_dirs(
+        first_dir_list, second_dir_path):
+    """Ensure that two dir's file counts match.
```

Comment:
This is a bit misleading, it should be "Ensure that the total count of files in all directories in the first list matches the count of files in the second directory."

Alternatively, you could generalize the existing function: compare_file_counts(first_dir_list, second_dir_list) and use 1-element lists as needed.

Reply:
Done.